mitodl/odl-video-service
# Fix issue with the Youtube play icon on mobile devices

#### What are the relevant tickets?
Fixes #669 

#### What's this PR do?
Forces the VideoJS play icon to display over Youtube's.  Previously it was not displaying on mobile devices, and the Youtube play icon wasn't working.

#### How should this be manually tested?
- Make one of your videos play back from Youtube: in admin, make the `Video` public, and add a `YoutubeVideo` object for it, with the Youtube ID = any Youtube video you want to play back, and with status = 'processed'

- Test that the Youtube video plays back within OVS by going to `http://localhost:8089/videos/<your_video_key>/embed` and `http://localhost:8089/videos/<your_video_key>/
`
- Create a local HTML page with that embedded video:

  ```
  <html>
    <body>
      <iframe src="http://localhost:8089/videos/<video_key>/embed/" width="560" height="315" frameborder="0" allowfullscreen></iframe>
    </body>
  </html>
  ```

- Serve the page locally via `python -m http.server` and load it in Chrome.  It should behave normally (play, pause, etc)
- Using the Chrome developer tools, use the device toolbar to emulate a Galaxy S5, iPhone, etc. You should see this:

  <img width="418" alt="screen shot 2018-07-27 at 2 42 35 pm" src="https://user-images.githubusercontent.com/187676/43340756-58ab8158-91ab-11e8-95b8-e15f0554ac25.png">

  and not this:

  
    <img width="400" alt="screen shot 2018-07-27 at 3 12 50 pm" src="https://user-images.githubusercontent.com/187676/43342128-a0e330f2-91af-11e8-896a-256c1f5a3a28.png">


  and the video should start playing when you click the center play button.

- Videos should also behave as before on standard detail and embed pages, with videos streaming from Cloudfront.
